ddtrace/_trace/context.py

```diff
@@ -94,7 +94,7 @@ def __init__(
             # DEV: A `forksafe.RLock` is not necessary here since Contexts
             # are recreated by the tracer after fork
             # https://github.com/DataDog/dd-trace-py/blob/a1932e8ddb704d259ea8a3188d30bf542f59fd8d/ddtrace/tracer.py#L489-L508
-            self._lock = threading.RLock()
+            self._lock = RLock()
```

Comment:
Do we need to update the comment here now that the assumption is the native lock will be forksafe?